<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QMainWindow" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QRadioButton" name="Valve2On">
       <property name="text">
        <string>On</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="Valve3Off">
       <property name="text">
        <string>Off</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="Valve4On">
       <property name="text">
        <string>On</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="Valve3On">
       <property name="text">
        <string>On</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QRadioButton" name="Valve5Off">
       <property name="text">
        <string>Off</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="Valve5On">
       <property name="text">
        <string>On</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QRadioButton" name="Valve4Off">
       <property name="text">
        <string>Off</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Valve 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Valve 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Valve 3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Valve 4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Valve 5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="Valve2Off">
       <property name="text">
        <string>Off</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="Valve1On">
       <property name="text">
        <string>On</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="Valve1Off">
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Pneumatics_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pneumatics</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QRadioButton" name="Ball1OffRbtn">
       <property name="text">
        <string>Close</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_9</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="Ball3OffRbtn">
       <property name="text">
        <string>Close</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_11</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="BV1lbl">
       <property name="text">
        <string>Valve 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="Ball1OnRbtn">
       <property name="text">
        <string>Open</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_9</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="Ball2OffRbtn">
       <property name="text">
        <string>Close</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_10</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="BV4lbl">
       <property name="text">
        <string>Valve 4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="BV3lbl">
       <property name="text">
        <string>Valve 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="Ball2OnRbtn">
       <property name="text">
        <string>Open</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_10</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="Ball4OnRbtn">
       <property name="text">
        <string>Open</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_12</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="Ball3OnRbtn">
       <property name="text">
        <string>Open</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_11</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="BV2lbl">
       <property name="text">
        <string>Valve 2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QRadioButton" name="Ball4OffRbtn">
       <property name="text">
        <string>Close</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_12</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QRadioButton" name="BVallPause">
       <property name="text">
        <string>Pause</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_9</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="BVallPause_2">
       <property name="text">
        <string>Pause</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_10</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="BVallPause_3">
       <property name="text">
        <string>Pause</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_11</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QRadioButton" name="BVallPause_4">
       <property name="text">
        <string>Pause</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_12</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Ball_valve_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ball Valves</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="2">
      <widget class="QRadioButton" name="Motor3_rev">
       <property name="text">
        <string>Reverse</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_8</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Motor2_label">
       <property name="text">
        <string>Motor 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="Motor1_speed">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Motor3_label">
       <property name="text">
        <string>Motor 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="Motor2_rev">
       <property name="text">
        <string>Reverse</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_7</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="send_speed1">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="Motor2_fwd">
       <property name="text">
        <string>Forward</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_7</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Motor1_label">
       <property name="text">
        <string>Motor 1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="Motor3_fwd">
       <property name="text">
        <string>Forward</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_8</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="send_speed3">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="Motor2_speed">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="Motor1_fwd">
       <property name="text">
        <string>Forward</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="Motor1_rev">
       <property name="text">
        <string>Reverse</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="Motor3_speed">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="send_speed2">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QRadioButton" name="Motor1Off">
       <property name="text">
        <string>Off</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="Motor2Off">
       <property name="text">
        <string>Off</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_7</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="Motor3Off">
       <property name="text">
        <string>Off</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_8</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Motor_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Motors</string>
    </property>
   </widget>
   <widget class="QLabel" name="MotorSpeedlbl">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_6"/>
  <buttongroup name="buttonGroup_7"/>
  <buttongroup name="buttonGroup_8"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_9"/>
  <buttongroup name="buttonGroup_12"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_11"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_5"/>
  <buttongroup name="buttonGroup_10"/>
 </buttongroups>
</ui>
